src/main/java/org/jenkinsci/plugins/docker/workflow/WithContainerStep.java

```diff
@@ -279,34 +277,8 @@ private static FilePath tempDir(FilePath ws) {
             this.toolName = toolName;
         }
 
-        private void stopContainer(StepContext context) {
-            Launcher launcher;
-            TaskListener listener;
-            try {
-                launcher = context.get(Launcher.class);
-                listener = context.get(TaskListener.class);
-            } catch (Exception x2) {
-                LOGGER.log(Level.WARNING, null, x2);
-                return;
```

Comment:
Simplifying error handling: if something fails during clean up, just let it be thrown up and fail the step.